config/locales/en.yml

```diff
@@ -1,3 +1,5 @@
 en:
   keywords:
     upload: "Upload"
+    no_keyword: "You don't have any keywords. Please upload some."
+    see_details: "See details"
```

Comment:
1. Avoid plural at all costs whenever possible. See https://nimblehq.co/compass/development/code-conventions/#convention-15
2. Prefer describing UI elements instead of using the current text as a translation key:

```
view_detail_link:  "See details"
```

Reply:
Fixed in d97f23927d8025c46757ebe18c9a1e3e1225b4da